renderdoc/driver/gl/official/cgl.h

```diff
@@ -10,40 +10,44 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-  
+
```

Comment:
Looks like this file was clang-format'd by accident?

Reply:
Sorry, my bad!  I'll fix this with only my changes.